oemoflex/tools/plots.py

```diff
@@ -153,6 +154,71 @@ def group_agg_by_column(df):
     return df_grouped
 
 
+def replace_near_zeros(df):
+    r"""
+    Due to numerical reasons, values are sometime really small, e.g. 1e-8, instead of zero.
+    All values which are smaller than a defined tolerance are replaced by 0.0.
+
+    Parameters
+    ---------------
+    df : pandas.DataFrame
+        Dataframe with data.
+
+    Returns
+    ----------
+    df : pandas.DataFrame
+        DataFrame with replaced near zeros.
+    """
+    tolerance = 1e-3
+    df[abs(df) < tolerance] = 0.0
+    return df
+
+
+def preprocessing(df, bus_name, demand_name):
```

Comment:
This function needs a more specific name.

Reply:
What do you think about "prepare_plotting_data" ? Or rather more specific?

Reply:
prepare_dispatch_data?

Reply:
Yes, I like that